### PR TITLE
[T4A4][T13-B2] Gao Yu

### PR DESCRIPTION
**Pull request by GaoYu-Karen**
Use ISP.

### CODE REVIEW

**Comment by GaoYu-Karen** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable{
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Address.java:12]: why public?

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    
+    default String getPrintableString() {
+        String combine = "";
+        String[] arg = null;
+        for (int i = 0; i < arg.length; i++) {
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Printable.java:8]: I wonder whether this compiles. Coz you haven't initialized the method local variable.  Its good that you try to do something new, but is that really required?

**Comment by KulaniT** [comment]
@GaoYu-Karen : Good try implementing ISP. You can also think another good place to put String getPrintableString(Printable... printables)

Please close the PR after reading comments

**Pull request closed without merging**